README.md

```diff
@@ -141,8 +141,8 @@ const catWatch = new ServiceClient({
         shouldRetry(err, req) {
             return (err && err.response && err.response.statusCode >= 500);
         },
-        onRetry(currentAttempt, err) {
-            console.log('Retry attempt #' + currentAttempt + 'due to ' + err);
+        onRetry(currentAttempt, err, req) {
+            console.log('Retry attempt #' + currentAttempt + ' for ' + req + ' due to ' + err);
```

Comment:
This will become `[object Object]`

Reply:
This was just an example for the readme, i can change it to req.path if you prefer?

Reply:
Yeah, or `JSON.stringify()`

Reply:
Fixed :)